packages/sdk/src/utils.ts

```diff
@@ -41,11 +38,10 @@ export const parseBoxToScreenCoords = ({
     .map((num) => parseFloat(num.trim()));
 
   const [x1, y1, x2 = x1, y2 = y1] = coords;
-  const [widthFactor, heightFactor] = factors;
 
   return {
-    x: Math.round(((x1 + x2) / 2) * screenWidth * widthFactor) / widthFactor,
-    y: Math.round(((y1 + y2) / 2) * screenHeight * heightFactor) / heightFactor,
+    x: Math.round(((x1 + x2) / 2) * screenWidth * 1000) / 1000,
+    y: Math.round(((y1 + y2) / 2) * screenHeight * 1000) / 1000,
   };
```

Comment:
Should the Action Parser section be changed? Currently, `factors` are still used in **coordinate mapping**, but we use `screenWidth` / `screenHeight` in  **coordinate restore** here.

```ts
          // Convert to float and scale
          const floatNumbers = numbers.map((num, idx) => {
            const factorIndex = idx % 2;
            return Number.parseFloat(num) / factors[factorIndex];
          });
```

Reply:
When calling this method, `x` and `y` are already ratios relative to physical pixels